### ISSUE TITLE
Locks up on Raspberry Pi

### ISSUE THREAD

**Issue by Willy65** [seandepagnier/statusbar_pi#16]
When I enable the module statusbar_pi on my Raspberry (if available), the plugin/options screen freezes.
It seems as if there is some Confirm button that isn't visible.
The enable tick-box works, also for other plugins, but the OK - Cancel - Apply buttons are unresponsive. Also the plugin window cannot be closed (by clicking X).

OpenCPN: Compiled from source as described at
https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn:developer_manual:developer_guide:compiling_linux:rpi2
and
https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn:developer_manual:developer_guide:compiling_linux
-> No compilation errors

Statusbar_pi: Compiled from source
root@opencpn:~# cd install/
root@opencpn:~# git clone git://github.com/seandepagnier/statusbar_pi.git
cmake and make
-> No compilation errors

Configuration:
Model: Raspberry Pi 3 Model B Rev 1.2
OS: Linux opencpn 4.19.97-v7+ #1294 SMP
GPS: Adafruit Ultimate GPS hat (serial) with PPS on ttyAMA0
RTC: Adafruit I2C
OpenGL: Enabled GL (Fake KMS)
Screen: Official Raspberry touchscreen 800x480 See https://www.raspberrypi.org/products/raspberry-pi-touch-display/
OpenCPN: Version 5.1.712
Statusbar_pi: Version 0.7 (2018-04-02)

The attached screenshot comes from a VNC session with higher resolution then the touchscreen. The behaviour is the same as on the physical touchscreen.

![opencpn_freeze](https://user-images.githubusercontent.com/16215539/88714445-20a28500-d11d-11ea-9067-b2612fec058c.png)

**Comment by Willy65** [comment]
[opencpn.log.txt](https://github.com/seandepagnier/statusbar_pi/files/5011298/opencpn.log.txt)
[opencpn.conf.txt](https://github.com/seandepagnier/statusbar_pi/files/5011300/opencpn.conf.txt)


Meanwhile I recompiled. Current version of OpenCPN is 5.2.0.
Also, I included libjpeg and libjasper in the compile (make clean, cmake, make && make install for both OpenCPN and statusbar_pi).
No change. As soon as the tab "Plugins" is selected, the system freezes.
The very first time that OpenCPN was started, the module statusbar_pi wasn't found and all was fine. Starting OpenCPN a 2nd time, the module statusbar_pi was found and OpenCPN freezes up again.

Attached opencpn.log and opencpn.conf from the ~/.opencpn directory.
